content/overview/user-interface/views.md

```diff
@@ -6,14 +6,22 @@ weight: 10
 author: "people"
 ---
 
-The functionality in darktable is separated into six different views:
+The functionality in darktable is delivered through two main views and four subsidiary views:
+
+**Main Views**
+<br>Switch between main views by clicking the view name at the top right of the window:
```

Comment:
Please avoid html (remove the `<br>`). If you must have a new paragraph you can use a line feed in markdown. Similarly below.

Reply:
Agreed